### DISCUSSION TITLE
FireFox PWA Introduces Malware + Error Occurs

### DISCUSSION THREAD

**Discussion by nanashinasubi** [filips123/PWAsForFirefox#479] [General]
I really wanted this application to work. However, every time I download the installation packages and run the system, Microsoft virus & threat protection identifies severe Trojan malware in the files. When the system removes the Trojan malware, the extension has an error and I cannot save any websites as PWAs. Any workarounds? 
![image](https://github.com/filips123/PWAsForFirefox/assets/164282731/7b21dafe-2423-493e-a14f-9e726082e934)
![image](https://github.com/filips123/PWAsForFirefox/assets/164282731/ef4cba67-96ab-464c-b1a5-1f93aed652e0)

**Comment by filips123** [comment]
It is a false positive. You need to update your malware definitions (and probably reinstall PWAsForFirefox). See more details in #480.
